src/http/client.rs

```diff
@@ -2572,12 +2572,46 @@ impl Http {
         wait: bool,
         files: Vec<CreateAttachment>,
         map: &impl serde::Serialize,
+    ) -> Result<Option<Message>> {
+        self.execute_webhook_(webhook_id, thread_id, token, wait, files, map, false).await
+    }
+
+    /// Same as [`Self::execute_webhook`] but allows sending non interactive components on non
+    /// app-owned webhooks.
+    ///
+    /// [Discord docs]: https://discord.com/developers/docs/resources/webhook#execute-webhook-query-string-params
```

Comment:
Should mention `[Discord docs]` somewhere, or else this link won't show. It is just defining that wherever `[Discord docs]` is used, it should refer to this link.